spec/tapioca/compilers/sorbet_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "spec_helper"
+
+class Tapioca::Compilers::SorbetSpec < Minitest::Spec
+  before do
+    @temp_env_value = T.let(nil, T.nilable(String))
+    @temp_env_value = ENV["TPC_SORBET_EXE"]
+  end
+
+  after do
+    ENV["TPC_SORBET_EXE"] = @temp_env_value
+  end
+
+  it("returns the value of TPC_SORBET_EXE if set") do
+    custom_path = 'bin/custom-sorbet-static'
+    ENV["TPC_SORBET_EXE"] = custom_path
+    sorbet_path = Tapioca::Compilers::Sorbet.sorbet_path
+    assert_equal(sorbet_path, custom_path)
+  end
+
+  it("returns the default sorbet path if TPC_SORBET_EXE is not set") do
```

Comment:
This is not exactly testing that the value is not set. It is testing the case that the value is set but it is set to an empty string. Can you add another test case for when the value of the env var is `nil` to test for env var not being set?